src/main/webapp/script.js

```diff
@@ -16,13 +16,35 @@ $('#myModal').on('shown.bs.modal', function () {
     $('#myInput').trigger('focus')
 });
 
-function initializeModal(pathId, itemId, itemName) {
-    console.log("initializing", arguments)
+function initializeModal(itemId, itemName) {
+    console.log("initializing", arguments);
 
     $("#feedback-title").text(itemName);
-    const form = $("#feedback");
+    $("#feedback-id").attr("value", itemId);
 }
 
-function submitFeedback() {
-  // TODO
-}
+async function submitFeedback() {
+    console.log("Sending the submission form info.");
+    const itemId = document.getElementById("feedback-id").value;
+    const pathId = document.getElementById("path-id").value;
+    const rating = document.getElementById("ratingValues").value;
+
+    console.log("TESTING VALUES: itemId " + itemId +
+    " pathId " + pathId + " rating " + rating);
+
+    //const form = $("#feedback");
+    const FD = new FormData();
+    FD.append("rating", rating);
+    FD.append("done", true);
```

Comment:
Apparently browsers (at least Chrome) encode this form as "multi-part" data, not the normal way that forms are encoded.

```
[INFO] ------WebKitFormBoundaryO3uYUUmZpMgBBcJK
[INFO] Content-Disposition: form-data; name="rating"
[INFO] 
[INFO] 1
[INFO] ------WebKitFormBoundaryO3uYUUmZpMgBBcJK
[INFO] Content-Disposition: form-data; name="done"
[INFO] 
[INFO] true
[INFO] ------WebKitFormBoundaryO3uYUUmZpMgBBcJK--
[INFO] 2020-04-28 09:09:47.590:WARN:oejs.HttpChannel:qtp107456312-26: /learning-path/23/item/16
```

The normal servlet request.getParameter(..) function doesn't know how to interpret this. ugh.

Reply:
For comparison, here's what the posted data looks like using `URLSearchParams`:

```
[INFO] {rating=[1],done=[true]}
```